packages/api/src/utils/client/httpClient.ts

```diff
@@ -115,7 +115,11 @@ export class HttpClient implements IHttpClient {
       }
       // De-duplicate by baseUrl, having two baseUrls with different token or timeouts does not make sense
       if (!this.urlsInits.some((opt) => opt.baseUrl === urlInit.baseUrl)) {
-        this.urlsInits.push({...urlInit, urlIndex: i} as UrlInitRequired);
+        this.urlsInits.push({
+          ...urlInit,
+          urlIndex: i,
+          printableUrl: toPrintableUrl(urlInit.baseUrl),
+        } as UrlInitRequired);
```

Comment:
why type cast here?

Reply:
type inference does not work properly, but figured if we explictly re-assing the base url it does, fixed by last commit